Create permissions.md (#10)
fytinnovations/oc-helpdesk: getting-started/permissions.md

---
sidebarDepth: 3
---

# Permissions

## Manage Helpdesk

> This permission will allow the user to the whole of helpdesk except assigning tickets to other backend users.

Reviewer: change the sentence to
This permission will allow the user to manage the entire helpdesk except assigning tickets to other backend users.

> This permission will allow the user to the whole of helpdesk except assigning tickets to other backend users.

Reviewer: ```suggestion
This permission will allow the user to the manage the entire helpdesk with the exception of assigning tickets to other  support staff.
```

## Assign Tickets

> Only users have this permissions would be able to assign tickets to other users.

Reviewer: ```suggestion
 Support staff with this permission have the ability to assign the tickets to other support staff 
```
